build_tools/packaging/linux/build_package.py

```diff
@@ -0,0 +1,497 @@
+#!/usr/bin/env python3
+"""Given ROCm artifacts directories, performs packaging to
+create RPM and DEB packages and upload to artifactory server
```

Comment:
I'd think you should put an SPDX header for the license of this (and all) scripts